RadixWallet/Clients/DeepLinkHandlerClient/DeepLinkHandlerClient+Test.swift

```diff
@@ -0,0 +1,18 @@
+// MARK: - DeepLinkHandlerClient + TestDependencyKey
+extension DeepLinkHandlerClient: TestDependencyKey {
+	public static let previewValue = Self.noop
+
+	public static let testValue = Self(
```

Comment:
I *think* this is not needed any more, if you use `@DependencyClient` macro on `DeepLinkHandlerClient`.